pac4j-stormpath/src/main/java/org/pac4j/stormpath/profile/StormpathProfile.java

```diff
@@ -0,0 +1,13 @@
+package org.pac4j.stormpath.profile;
+
+import org.pac4j.core.profile.CommonProfile;
+
+/**
+ * <p>The user profile returned from a Stormpath Authentication event.</p>
+ * @author Misagh Moayyed
+ * @since 1.8
+ */
+public class StormpathProfile extends CommonProfile {
```

Comment:
We could maybe go further here by overriding the appropriate methods of the `CommonProfile` and by getting the appropriate properties. Maybe the `getDisplayName` should return the "fullName" attribute, the `getUsername` method should return the "surName"...